### PR TITLE
fix: fix process descriptions and layouts

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8412bb9337cd5016ea5e9dc79c620270d976b7e1
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@pmuir snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1150-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/shared/dummy.service.ts [hunk 1]
```diff
@@ -106,21 +106,19 @@ export class DummyService {
...
     {
       name: 'Agile',
-      description: `Agile is self-organizing cross-functional teams, that focus on planning, evolutionary development, 
-      early delivery, and continuous improvement.`
+      description: `Agile-based planning (more generic than Scrum)`
     },
     {
       name: 'Scrum',
-      description: `Scrum is an iterative and incremental agile software development framework for managing product development.`
+      description: `Scrum-based planning.`
```
Comment (joshuawilson) [inline src/app/shared/dummy.service.ts]: @pmuir Please update this line to the new line in the doc that @catrobson updated.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 04eb10e2c7e97007c8839621ebaf9f95b2a52678
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 743cd990be247943f68ef44673e52871e564a584
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/app/shared/dummy.service.ts (hunk 1 above)
Comment (pmuir) [inline src/app/shared/dummy.service.ts]: Done

**Comment by fabric8cd** [comment]
@pmuir snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1150-fabric8-ui.int.rdu2c.fabric8.io

**Pull request merged**